### ISSUE TITLE
Fullscreenmode for secondary editor

### ISSUE THREAD

**Issue by mhsdesign** [neos/neos-ui#3095]
### Description

currently we have the fullscreen button, but its only meant for the inline editable content:

https://user-images.githubusercontent.com/85400359/161242164-2014af45-0764-4c2a-a0c0-045f3cd82cb1.mp4

so clicking on it with changes, wouldnt work:

https://user-images.githubusercontent.com/85400359/161242398-da1a6e6f-c131-4703-a634-def56e6c04d2.mp4

### Steps to Reproduce

1. Open an HTML element of the Neos.Demo and edit it.
2. Click on the full screen button.

#### Expected behavior

that there this button would be also working for secondary editors - or that this whole toolbar is switched or hidden.

#### Actual behavior

the button is pretty stubborn and always wants me to focus only on the editable content ^^
- which *is* also fine - but i wanted to discuss that.

### Affected Versions
UI:
since ever.

### Afterthought

i really like the idea of switching the toolbar, then we can also put this X button in the toolbar:
![image](https://user-images.githubusercontent.com/85400359/161243438-a7fc1629-794e-4998-bcb9-b3fbebe8e60e.png)

**Comment by Sebobo** [comment]
In the media ui I toggle the left sidebar when opening the secondary editor. It's a bit hacky and when reloading while the editor is open, the left sidebar is still hidden... 
So a better solution would be cool.

**Comment by mhsdesign** [comment]
i took a look at the architecture, to implement this idea we would need some refactoring, as currently the left `<Inspector>` makes just some `<Portal>` which is completely absolute positioned ...

![image](https://user-images.githubusercontent.com/85400359/161247910-6fe81aa6-99a4-4c0c-9198-69d3bb09a3cc.png)

**Comment by mhsdesign** [comment]
@Sebobo so your ideal fullscreen would still include the right inspector? (or i guess you only hid the left side since only that is possible...)

**Comment by Sebobo** [comment]
I would like to keep the top area and optionally the right sidebar, because it might be helpful for context of the secondary editor.

**Comment by mhsdesign** [comment]
i solved it for now by using
`htmlElement.requestFullscreen()`
to also get native fullscreen (triggered via shortcode F11)

**Comment by danielkestler** [comment]
Thought about it some years ago. Probably something in the line of this?

<img width="1488" alt="Bildschirmfoto 2022-07-07 um 08 31" src="https://user-images.githubusercontent.com/3976405/177724540-83139849-3262-4e1f-96b4-d9ac851fa131.png">

**Comment by mhsdesign** [comment]
![image](https://user-images.githubusercontent.com/85400359/190849819-9efd8f00-8de3-4afa-b02d-3b9a34c48b90.png)

I would prefer it like this:
And make it configurable, what happens with your editor when clicking that button:

Options:
A. - Like I suggested `secondaryInspector.requestFullscreen()`
B. - Keep the 40px secondary toolbar at the top and just fold everything back.
C. - Keep the 40px secondary toolbar at the top and just fold everything back but the inspector like @Sebobo suggested.

secondary toolbar:
![image](https://user-images.githubusercontent.com/85400359/190850117-eabfe3c0-6a12-4b2b-93f9-9ccf8dbf1992.png)

--------

I also like what i hacked here ^^
![image](https://user-images.githubusercontent.com/85400359/190850225-28595f0e-ebf3-44c7-ad32-007baa809683.png)
Basically i moved the editor up a bit (40px), and made the toolbar my own.

**Comment by mhsdesign** [comment]
@danielkestler that seems like an entirely different feature ^^
